src/traces/indicator/defaults.js

```diff
@@ -0,0 +1,162 @@
+    function coerce(attr, dflt) {
+        return Lib.coerce(traceIn, traceOut, attributes, attr, dflt);
+    }
+
+    handleDomainDefaults(traceOut, layout, coerce);
+
+    // Mode
+    coerce('mode');
+    traceOut._hasNumber = traceOut.mode.indexOf('number') !== -1;
+    traceOut._hasDelta = traceOut.mode.indexOf('delta') !== -1;
+    traceOut._hasGauge = traceOut.mode.indexOf('gauge') !== -1;
+
+    coerce('value');
+    coerce('vmin');
+    coerce('vmax', 1.5 * traceOut.value);
+
+    // Number attributes
+    var auto = [];
+    var bignumberFontSize;
+    if(traceOut._hasNumber) {
+        coerce('number.valueformat');
+        coerce('number.font.color', layout.font.color);
+        coerce('number.font.family', layout.font.family);
+        coerce('number.font.size', 'auto');
+        if(traceOut.number.font.size === 'auto') {
+            traceOut.number.font.size = cn.defaultNumberFontSize;
+            auto[0] = true;
+        }
+        coerce('number.suffix');
+        bignumberFontSize = traceOut.number.font.size;
+    }
+
+    // delta attributes
+    var deltaFontSize;
+    if(traceOut._hasDelta) {
+        coerce('delta.font.color', layout.font.color);
+        coerce('delta.font.family', layout.font.family);
+        coerce('delta.font.size', 'auto');
+        if(traceOut.delta.font.size === 'auto') {
+            traceOut.delta.font.size = (traceOut._hasNumber ? 0.5 : 1) * (bignumberFontSize || cn.defaultNumberFontSize);
+            auto[1] = true;
+        }
+        coerce('delta.reference', traceOut.value);
+        coerce('delta.relative');
+        coerce('delta.valueformat', traceOut.delta.relative ? '2%' : '.3s');
+        coerce('delta.increasing.symbol');
+        coerce('delta.increasing.color');
+        coerce('delta.decreasing.symbol');
+        coerce('delta.decreasing.color');
+        coerce('delta.position');
+        deltaFontSize = traceOut.delta.font.size;
+    }
+    traceOut._scaleNumbers = (!traceOut._hasNumber || auto[0]) && (!traceOut._hasDelta || auto[1]) || false;
+
+    // Title attributes
+    coerce('title.font.color', layout.font.color);
+    coerce('title.font.family', layout.font.family);
+    coerce('title.font.size', 0.25 * (bignumberFontSize || deltaFontSize || cn.defaultNumberFontSize));
+    coerce('title.text');
+
+    // Gauge attributes
+    var gaugeIn, gaugeOut, axisIn, axisOut;
+    function coerceGauge(attr, dflt) {
+        return Lib.coerce(gaugeIn, gaugeOut, attributes.gauge, attr, dflt);
+    }
+    function coerceGaugeAxis(attr, dflt) {
+        return Lib.coerce(axisIn, axisOut, attributes.gauge.axis, attr, dflt);
+    }
+    if(traceOut._hasGauge) {
+        gaugeIn = traceIn.gauge;
+        gaugeOut = Template.newContainer(traceOut, 'gauge');
+        coerceGauge('shape');
+        var isBullet = traceOut._isBullet = traceOut.gauge.shape === 'bullet';
+        if(!isBullet) {
+            coerce('title.align', 'center');
```

Comment:
Doesn't the current behaviour for bullets have essentially `title.align: 'right'`?

Reply:
> Doesn't the current behaviour for bullets have essentially `title.align: 'right'`?

They do yes. However, it cannot be changed. That's why I thought it shouldn't be coerced. Should I coerce it?

Reply:
Could it support `align: 'center'` and `align: 'left'` eventually (i.e. not in this PR)?

Reply:
> Could it support `align: 'center'` and `align: 'left'` eventually (i.e. not in this PR)?

My thought was that because the title is outside the domain in `bullet`, the notion of alignment is not well-defined: it's unclear how much whitespace we have on the left (ie. maybe there's another plot there).